docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md

```diff
@@ -139,113 +155,3 @@ The following functions are also available when expanding labels and annotations
 | [args](#args)                             | []interface{}                                                | map[string]interface{} | Converts a list of objects to a map with keys, for example, arg0, arg1. Use this function to pass multiple arguments to templates.          |
 | [externalURL](#externalurl)               | nothing                                                      | string                 | Returns a string representing the external URL.                                                                                             |
 | [pathPrefix](#pathprefix)                 | nothing                                                      | string                 | Returns the path of the external URL.                                                                                                       |
-
-#### humanize
```

Comment:
Has this been moved to another documentation page?

Reply:
I would consider it a duplicate of the table above it. I think when I last edited this page I copied the functions verbatim from somewhere else in the documentation without editing it, and so we have 1. a table that lists all functions 2. inline text that lists all functions below the table. We could perhaps move some of the examples up into the table though.

Reply:
Putting some of the examples into the table isn't great because the page isn't wide enough. We could make them separate topics with individual examples in a follow up pull request?

Reply:
Putting some of the examples into the table isn't great because the page isn't wide enough. We could make them separate topics with individual examples in a follow up pull request?

Reply:
I just pushed those changes 🙂